br/pkg/lightning/restore/meta_manager.go

```diff
@@ -1068,15 +1071,34 @@ func (b singleMgrBuilder) TableMetaMgr(tr *TableRestore) tableMetaMgr {
 }
 
 type singleTaskMetaMgr struct {
-	pd *pdutil.PdController
+	pd           *pdutil.PdController
+	taskID       int64
+	initialized  bool
+	sourceBytes  uint64
+	clusterAvail uint64
 }
 
 func (m *singleTaskMetaMgr) InitTask(ctx context.Context, source int64) error {
+	m.sourceBytes = uint64(source)
+	m.initialized = true
 	return nil
 }
 
 func (m *singleTaskMetaMgr) CheckTasksExclusively(ctx context.Context, action func(tasks []taskMeta) ([]taskMeta, error)) error {
-	_, err := action(nil)
+	newTasks, err := action([]taskMeta{
```

Comment:
seems changes in this file is not related to the fixing.

Reply:
When checking cluster resource, `sourceBytes` need to read by `CheckTasksExclusively`.
https://github.com/pingcap/tidb/blob/b7f26143e500379fec94c286badcac26bdfc90d6/br/pkg/lightning/restore/check_info.go#L127